### ISSUE TITLE
README: explain "insert OAuth info"

### ISSUE THREAD

**Issue by simon04** [danmichaelo/croptool#62]
I would find a more detailed explanation of "insert OAuth info" helpful. For instance – as far as I understand:

> Apply for a [owner-only](https://www.mediawiki.org/wiki/OAuth/Owner-only_consumers) OAuth consumer token at https://meta.wikimedia.org/wiki/Special:OAuthConsumerRegistration/propose and fill out `consumerKey`, `consumerSecret` in the `config.ini`

**Comment by danmichaelo** [comment]
Right.. didn't think about that. And the problem is that it's kinda "expensive" to create a new consumer, since each registration is reviewed manually.. at least it was manual (and took a few days) the last time I had to register.

**Comment by ricordisamoa** [comment]
https://gerrit.wikimedia.org/r/255488
> An owner-only consumer is automatically approved, automatically
> accepted by the owner, cannot be used by anyone other than the owner,
> cannot use the usual authorization flow, and does not tag changes.

**Comment by danmichaelo** [comment]
Good news!

**Comment by lucaswerkmeister** [comment]
Owner-only consumers have a different “flow” than regular consumers, so I don’t think they’ll work. But the good news is that regular consumers will work for their owner as soon as they’re created, and will continue to work even after they’ve been rejected by an OAuth admin. So you should be able to request an OAuth consumer and then immediately use it, and tell the admins “don’t bother accepting this one, it’s only for development anyways” :)

**Comment by lucaswerkmeister** [comment]
Sorry, that’s incorrect – the owner can use the consumer while it hasn’t been approved yet, but not after it’s been rejected. That’s a shame.
